module-2/backend/week_one.md

```diff
@@ -5,30 +5,58 @@ Fork this respository. Answer the questions to the best of your ability. Try to
 Note: When you're done, submit a PR. 
 
 1. List the five common HTTP verbs and what the purpose is of each verb.
+get, puts, delete, post, patch
 2. What is Sinatra?
+sinatra is the framework we use to create web apps
 4. What is MVC?
+model view controller

```

Comment:
Yup! Recommend you also be familiar with the responsibilities of each of these pieces.